openfeature-provider/pom.xml

```diff
@@ -10,11 +10,13 @@
   <artifactId>openfeature-provider</artifactId>
 
   <dependencies>
+    <!-- x-release-please-start-version -->
```

Comment:
Would love to have this extracted into its own PR so we can merge it quick and easy 👍